lib/active_fedora/attributes.rb

```diff
@@ -126,6 +126,11 @@ def system_attributes
         ['has_model', 'create_date', 'modified_date']
       end
 
+      # Attributes inherited from Fedora in ActiveFedora::FedoraAttributes
+      def inheritable_attributes
```

Comment:
Yes, it's not the attributes that are inheritable here, it is the methods that are (similar to class_attribute) https://github.com/projecthydra/active_fedora/blob/80b830ee76ce9407058250a00f83b570a46aa7ee/lib/active_fedora/inheritable_accessors.rb#L8  How about `attributes_with_defaults` ?